archlinux/PKGBUILD

```diff
@@ -23,10 +23,19 @@ backup=()
 options=('staticlibs')
 changelog=
 
-source=(PKGBUILD-initcpio-hook.sh PKGBUILD-initcpio-install.sh PKGBUILD-qubes-prepare-vm-kernel.sh)
+source=(PKGBUILD-initcpio-hook.sh
+	PKGBUILD-initcpio-install.sh
+	PKGBUILD-qubes-prepare-vm-kernel.sh
+	PKGBUILD-qubes-prepare-vm-kernel.hook
+	)
+
+md5sums=(e88b50aed7d28087968da728ec1dc2b6
+	3d204b0d8acda9cf3ea5ebd5334a04d6
+	6b33d5cdacf9873d270bf785e3c4f39b
+	8e0754a613a2fa0a578361455d225b48
+	)
```

Comment:
Actually, I started to disable verification in all archlinux packages, because qubes-builder already check the integrity of scripts.